tests/foreman/api/test_activationkey.py

```diff
@@ -126,6 +130,20 @@ def test_negative_create_3(self):
                         unlimited_content_hosts=True,
                     ).create()
 
+    def test_negative_create_4(self):
```

Comment:
what about naming this as test_negative_create_name? We are trying to have better naming for the test methods and  with that just reading the name we now that we are passing a invalid name on creation (negative + name + create)